tests/integration/devfile/cmd_devfile_catalog_test.go

```diff
@@ -53,6 +53,7 @@ var _ = Describe("odo devfile catalog command tests", func() {
 				"NAME",
 				"java-spring-boot",
 				"openLiberty",
+				"quarkus",
```

Comment:
I think it should also be checked in https://github.com/openshift/odo/pull/3172/files#diff-318517042700080474192fc7dc04e456L67-L76

Reply:
@prietyc123 Added